playbooks/templates/k8s/jail/matomo-jail.yml.j2

```diff
@@ -167,6 +167,20 @@ data:
         ## only allow accessing the following php files
         location ~ ^/(index|plugins/HeatmapSessionRecording/configs)\.php {
 {% endif %}
+
+{% if matomo_name == 'matomo-api' %}
+          access_by_lua_block {
+            local opts = {
+              defaultCsp = "https://{{ wai_hostname[matomo_ingress_fqdn_index] }}.{{ wai_base_domain[item] }}",
+              redis = {
+                url = "redis://{{ redis_password.application_redis_master[item] }}@{{ application-redis-master }}:6379/10",
```

Comment:
Credo che `{{ application-redis-master }}` non sia una variabile.

L'indice del DB dovrebbe essere configurabile.

Reply:
In merito a `{{ application-redis-master }}` hai ragione.
Per il database allora lo mettiamo anche su portal o solo su questo database ?

Reply:
Hai ragione, mettiamo anche su portal.